helper/src/components/previewDialog.js

```diff
@@ -1,34 +1,36 @@
 import React from "react";
 import { useBoolean } from "@fluentui/react-hooks";
-import { Dialog, DialogType, DialogFooter } from '@fluentui/react/lib/Dialog';
-import { PrimaryButton,DefaultButton } from '@fluentui/react/lib/Button';
+import { Dialog, DialogType, DialogFooter } from "@fluentui/react/lib/Dialog";
+import { PrimaryButton, DefaultButton } from "@fluentui/react/lib/Button";
 
-export function PreviewDialog({previewLink}) {
-
-  const [hideDialog, { toggle: toggleHideDialog }] = useBoolean(false);
+export function PreviewDialog({ previewLink, onClose, show}) {
+  // const [hideDialog, { toggle: toggleHideDialog }] = useBoolean(false);
 
   const dialogContentProps = {
     type: DialogType.normal,
-    title: 'Preview Feature',
-    closeButtonAriaLabel: 'Close',
+    title: "Preview Feature",
+    closeButtonAriaLabel: "Close",
     subText: `Review the instructions on this page ${previewLink} to enable the feature `,
   };
 
   function _openLink() {
-    window.open(`${previewLink}`, '_blank', 'noreferrer');
+    window.open(`${previewLink}`, "_blank", "noreferrer");
   }
 
+  if(!show){
+    return null;
+}
   return (
-    <div>
-      <Dialog
-      hidden={hideDialog}
-      onDismiss={toggleHideDialog}
-      dialogContentProps={dialogContentProps}>
-      <DialogFooter>
-        <PrimaryButton onClick={toggleHideDialog} text="Close" />
-        <DefaultButton onClick={_openLink} text="Open Link in New Tab" />
```

Comment:
I dont think we need a button here, just a <a> clickable link

Reply:
![image](https://github.com/Azure/AKS-Construction/assets/3062772/8429b2e0-c15d-4d29-b541-5e1d1cf25c53)
 new look